core/base/compactTriangulation/CompactTriangulation.h

```diff
@@ -0,0 +1,4214 @@
+/// \ingroup base
+/// \class ttk::CompactTriangulation
+/// \author Guoxi Liu <guoxil@g.clemson.edu>
+/// \date November 2021.
+///
+/// \brief CompactTriangulation is a class implemented based on the
+/// TopoCluster data structure, which is a localized data structure for
+/// simplicial meshes.
+/// The key idea of TopoCluster is to subdivide the simplicial mesh into
+/// clusters. Then, the connectivity information is computed locally for each
+/// cluster and discarded when it is no longer needed. The simplicial mesh needs
+/// to be subdivided before using TopoCluster, i.e., there has to be a scalar
+/// field named "ttkCompactTriangulationIndex" to denote the cluster index of
+/// each vertex. Note Topocluster will reindex the simplices based on the
+/// clustering input array.
+/// \b Related \b publications \n
+/// "TopoCluster: A Localized Data Structure for Topology-based Visualization"
```

Comment:
don't forget to also add that reference:
- in the corresponding vtk header
- in the paraview xml file

Reply:
solved